tests/by-util/test_touch.rs

```diff
@@ -118,6 +118,45 @@ fn test_touch_set_mdhms_time() {
     assert_eq!(mtime.unix_seconds() - start_of_year.unix_seconds(), 45296);
 }
 
+#[test]
+fn test_touch_2_digit_years_68() {
+    // 68 and before are 20xx
+    let (at, mut ucmd) = at_and_ucmd!();
+    let file = "test_touch_set_two_digit_68_time";
+
+    ucmd.args(&["-t", "6801010000", file])
+        .succeeds()
+        .no_stderr();
```

Comment:
A detail: I would use `no_output` to indicate that no output is expected.
```suggestion
        .no_output();
```